examples/player/simid_player.js

```diff
@@ -609,14 +595,82 @@ class SimidPlayer {
     console.log('The creative has asked for the player to ping ' + requestedUrlArray);
   }
 
+  /**
+   * Called when video playback is complete.
+   * @private
+   */
+  videoComplete() {
```

Comment:
is this function called anymore?  Looks like you commented out the code that calls this.

Reply:
uncommented